microprofile/WFLY-12737_quickstart-for-mp-health-2.2.adoc

```diff
@@ -0,0 +1,68 @@
+= WFLY-12737 Quickstart for MP Health 2.2
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This RFE provides a new quickstart and guide for MicroProfile Health 2.2 use in WildFly /
+JBoss EAP. The example will cover all basic functionalities of MicroProfile Health 2.2,
+namely: liveness and readiness checks, the basic usage of the `HealthCheckResponseBuilder`,
+and the inclusion of user specific data in the health check responses.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-12737[WFLY-12737]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1368[EAP7-1368] (internal)
+
+=== Dev Contacts
+
+* mailto:mstefank@redhat.com[Martin Stefanko]
+
+=== QE Contacts
+
+* mailto:szhantem@redhat.com[Sultan Zhantemirov]
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+[x] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* Quickstarts
+
+=== Other Interested Projects
+
```

Comment:
Just asking - I see `resteasy` is used in this QS? Would this information be relevant to this proposal? WDYT?

Reply:
only for testing to invoke /health endpoints. I find JAX-RS client clearer to use that plain HTTP. I don't think that needs to be mentioned.

Reply:
Yes maybe your right that it isn't relevant here, although I'd add a mention to it below, in the Test Plan" section, where you're providing some high level detail of involved components, e.g. Arquillian.

Reply:
I added the mention of resteasy to the test plan section.